### PR TITLE
Propagate changes from libsonata ElementReport

### PR DESCRIPTION
**Pull request by tomdele**
preparation for the new libsonata reports

### CODE REVIEW

**Comment by tomdele** [comment]
This prepares the bluepysnap for the libsonata bump. 
All tests are passing locally with the https://github.com/BlueBrain/libsonata/pull/93

**Comment by tomdele** [comment]
If I am lucky enought the changes for the numpy array will be handled seamlessly with these changes ....

**Comment by tomdele** [comment]
@asanin-epfl I will need your review when this is finished. 
This is to handle the new version of the libsonata report outputs. I will ping you when I need your review on this. I don t think I will do drastic changes so can already have a quick look.

**Comment by tomdele** [comment]
The CI cannot pass due to an unreleased version of libsonata (0.1.4)
https://github.com/BlueBrain/libsonata/pull/96

**Comment by tomdele** [comment]
Need to remove the fix_libsonata when this change is pulled out from libsonata:
https://github.com/BlueBrain/snap/pull/62/files#diff-67a29d152e00a31a251cd47da15b2e55R300

**Comment by asanin-epfl** [comment]
lgtm to me.

**Comment by tomdele** [comment]
Thx @asanin-epfl ! 
I will ping you again when all the changes will be done and when I will create a real PR.

**Comment by tomdele** [comment]
retest this please

**Comment by tomdele** [comment]
@asanin-epfl you can review.

**Comment by asanin-epfl** [comment]
ok, starting

**Comment by tomdele** [comment]
Thanks a lot.

File: bluepysnap/frame_report.py [hunk 1]
```diff
@@ -98,22 +97,36 @@ def get(self, group=None, t_start=None, t_stop=None):
...
 
-        view = self._frame_population.get(node_ids=ids, tstart=t_start, tstop=t_stop)
-        if not view.data:
+        if len(view.ids) == 0:
             return pd.DataFrame()
-        res = pd.DataFrame(data=view.data, index=view.index)
+
+        res = pd.DataFrame(data=view.data,
+                           columns=pd.MultiIndex.from_arrays(np.asarray(view.ids).T),
+                           index=view.times).sort_index(axis=1)
```
Comment (asanin-epfl) [inline bluepysnap/frame_report.py:111]: `view.times` are always guaranteed to be present?

Comment (tomdele) [reply bluepysnap/frame_report.py:111]: yes ! 
the view always contains the 3 objects

**Comment by tomdele** [comment]
thank you for the review @asanin-epfl !

**Pull request merged**